app/src/main/java/com/github/geohunt/app/ui/settings/RadioItem.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.geohunt.app.ui.settings
+
+import androidx.compose.material.Icon
+import androidx.compose.material.RadioButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import com.alorma.compose.settings.ui.SettingsMenuLink
+
+@Composable
+fun <T> RadioItem(
+    title: String,
+    value: T,
+    isSelected: Boolean,
+    onSelect: (T) -> Any,
+    modifier: Modifier = Modifier,
+    icon: ImageVector? = null,
+    subtitle: String? = null,
+    disabled: Boolean = false
+) {
```

Comment:
Jetpack compose don't have radio button ???? Kinda sad really

Reply:
No no, it does has one. However, not a stylized radio button as above. :(

Reply:
:'(